components/attempt.js

```diff
@@ -10,6 +10,9 @@ class Attempt extends React.Component {
       value: '',
       minLength: 2
     };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleEnter = this.handleEnter.bind(this);
```

Comment:
yup, you're right! this is a far better approach. however did you consider using the bindAll function from underscore.js?